themes/default/content/testimonials/_index.md

```diff
@@ -0,0 +1,161 @@
+    -
+      message: "Ok, Pulumi just got even more real. With “pulumi convert” and CrossCode support in YAML, it just overtook HCL and Terraform in position #1 for me. Sorry HashiCorp Pulumi is now in the lead."
+      author: Eric Hendrickson, CTO Provisions Group
+      link: https://www.linkedin.com/feed/update/urn:li:activity:6993614249331765250?commentUrn=urn%3Ali%3Acomment%3A%28activity%3A6993614249331765250%2C6993718113632411648%29
+      source: linkedin
+    -
+      message: "Continuing on my thread about @PulumiCorp from a while ago: holy shit I am a convert. I needed to setup a staging environment that was mostly identical to prod, and once I trued up our Pulumi stack with AWS, it took minutes to do this. How have I lived without this until now?"
+      author: "@krangarajan\n"
+      link: https://twitter.com/krangarajan/status/1580618068203479040
+      source: twitter
+    -
+      message: Pulumi is better than Terraform. Hands down. It has all the features of Terraform, but written with the simplicity and power of a fully featured programming language.
+      author: Christopher Lenard
+      link: https://medium.com/@christopherlenard/the-benefits-of-moving-from-terraform-to-pulumi-7e01a3ab8f43
+      source: blog
+      featured: true
+    -
+      message: "Having used Pulumi the last 2 years after switching from Terraform I want to share some of my experience by comparing both infrastructure as code tools and explain why I prefer Pulumi in most situations. In general, I have successfully provisioned infrastructure with both tools, but from my experience Pulumi makes it easier for me. (click to read blog post)"
+      author: Matthias Müller
+      link: https://www.novatec-gmbh.de/en/blog/why-you-should-consider-pulumi-over-terraform-for-your-next-project-pt-1/
+      source: blog
+    -
+      message: "Pulumi is awesome. This is the 3rd company I''ve introduced to Pulumi. Eventually I''ll get the TF stuff here moved over"
+      author: Derek Beattie, Sr Software Engineer, Jetlinx
+      link: https://docs.google.com/presentation/d/1adJEZE1iY0P6ZRT7xknLq_R3M-eb6m5dubXII-AFxlc/edit#slide=id.g15d990b8a01_8_589
+      source: email
+    -
+      message: "I want to thank you and your team for an amazing tool. I am in the starting stages of an AWS modernization effort with my current employer and part of that will be IaC with Pulumi. Great product and exciting vision for the future of infrastructure."
+      author: Travis
+      link: joe
+      source: email
+      featured: true
+    -
+      message: "Hey Joe!\n\nI'm a big fan so far! Love being able to work in C# instead of yaml. Type safety, intellisense, class abstractions, etc all feel great.\n\nI was using it for allowing for a quick setup of Keycloak on a local box that clones down my repo aand did run into a weird snag with setting up scopes, but not pulumi based as far as I can tell!\n\nRegardless, it's been pretty fun and I'm not even an infra guy"
```

Comment:
```suggestion
      message: "I'm a big fan so far! Love being able to work in C# instead of yaml. Type safety, intellisense, class abstractions, etc all feel great.\n\nI was using it for allowing for a quick setup of Keycloak on a local box that clones down my repo aand did run into a weird snag with setting up scopes, but not pulumi based as far as I can tell!\n\nRegardless, it's been pretty fun and I'm not even an infra guy"
```